lib/adyen/configuration.rb

```diff
@@ -105,6 +105,28 @@ def autodetect_environment
   # @return [String]
   attr_accessor :ipn_password
 
+  # The username that’s used to submit payouts. It should look
+  # something like ‘storePayout@AndyInc.SuperShop+’
+  #
+  # @return [String]
+  attr_accessor :payout_store_username
+
+  # Password used to store payout together with '+payout_review_username+' configuration attribute.
+  #
+  # @return [String]
+  attr_accessor :payout_store_password
+
+  # The username that’s used to review payouts. It should look
+  # something like reviewPayout@AndyInc.SuperShop+’
+  #
+  # @return [String]
+  attr_accessor :payout_review_username
+
+  # Password used to review payout together with '+payout_review_username+' configuration attribute.
+  #
+  # @return [String]
+  attr_accessor :payout_review_password
+
```

Comment:
I am not sure if this library even needs to know about these. If you want a singleton in your app, it's a one liner. The only reason I included a singleton was that previously, one user would allow you to do everything with the API. If that is no longer the case, I rather promote people instantiating their own clients, based on how they have set up their permissions - which can be different for everybody.

E.g.:
```
AdyenPayoutStoreClient = Adyen::Client.new(env, 'user', 'pass')
AdyenPayoutReviewClient = Adyen::Client.new(env, 'otheruser', 'otherpass')
```